### PR TITLE
Add functionality to bottom nav

### PR DESCRIPTION
**Pull request by Caruki**
Change bottom nav to a tab navigation with styled svgs

(no live preview available since the bug in the authentication is still not fixed in this PR)

### CODE REVIEW

File: client/src/components/BottomNav.js [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
-import RecsIconClicked from '../assets/recs-clicked-icon.svg';
+import { NavLink } from 'react-router-dom';
 
-const NavContainer = styled.div`
+const Container = styled.nav`
   display: flex;
   justify-content: center;
   width: 100%;
   flex: 1 0 10%;
+  list-style: none;
```
Comment (marwinburesch) [inline client/src/components/BottomNav.js:11]: do you need this? I don't see any lists in the dom

File: client/src/components/BottomNav.js [hunk 2]
```diff
@@ -24,78 +23,44 @@ const ButtonContainer = styled.div`
   flex: 1 0;
   border-radius: 25px 25px 0px 0px;
   background-color: rgba(14, 5, 46, 0.6);
+  cursor: pointer;
+  text-decoration: none;
 `;
 
-const Button = styled.button`
+const ListItem = styled.div`
```
Comment (marwinburesch) [inline client/src/components/BottomNav.js:30]: so it's a `<div>` disguised as a `ListItem`? 🧐

File: client/src/layoutRoutes/HeaderBottomNavLayoutRoute.js [hunk 3]
```diff
@@ -25,7 +29,22 @@ const HeaderBottomNavLayoutRoute = ({ component: Component, ...rest }) => {
           <MainContainer>
             <Component {...matchProps} />
           </MainContainer>
-          <BottomNav />
+          <BottomNav
+            links={[
+              {
+                label: 'Popular',
+                Icon: Popular,
```
Comment (marwinburesch) [inline client/src/layoutRoutes/HeaderBottomNavLayoutRoute.js:36]: is the uppercase "I" intentional?

File: client/src/layoutRoutes/HeaderBottomNavLayoutRoute.js [hunk 4]
```diff
@@ -25,7 +29,22 @@ const HeaderBottomNavLayoutRoute = ({ component: Component, ...rest }) => {
           <MainContainer>
             <Component {...matchProps} />
           </MainContainer>
-          <BottomNav />
+          <BottomNav
+            links={[
```
Comment (marwinburesch) [inline client/src/layoutRoutes/HeaderBottomNavLayoutRoute.js:33]: very cool approach btw :)

Comment (Caruki) [reply client/src/layoutRoutes/HeaderBottomNavLayoutRoute.js:33]: gotta admit I copied this from the others who worked up this approach with Leon 😉 so I cheated.. that's why it resulted also in the other stuff you commented 🙈

**Pull request merged**